src/goto-checker/solver_factory.cpp

```diff
@@ -411,7 +411,6 @@ void solver_factoryt::no_incremental_check()
 {
   const bool all_properties = options.get_bool_option("all-properties");
   const bool cover = options.is_set("cover");
-  const bool incremental_check = options.is_set("incremental-check");
```

Comment:
This should be `incremental-loop`.

Reply:
Ok. I have removed this deletion from this PR and pushed a fix with a regression test instead.